codegen_sources/test_generation/create_tests.py

```diff
@@ -39,6 +39,7 @@
 
 MUTATION_SCORE_CUTOFF = 0.9
 MAX_JAVA_MEM = 4096
+CPUS_PER_TASK=80
```

Comment:
What would you think of replacing this constant with `multiprocessing.cpu_count()` so that it adapts to the number of CPUs on the machine? (also add import multiprocessing above)

```suggestion
CPUS_PER_TASK=multiprocessing.cpu_count()
```